lib/pagseguro/phones.rb

```diff
@@ -0,0 +1,26 @@
+module PagSeguro
+  class Phones
+    extend Forwardable
+    include Enumerable
+    include Extensions::EnsureType
+
+    def_delegators :@store, :size, :clear, :empty?, :any?, :each
+
+    def initialize
+      @store = []
+    end
+
+    def << (phone)
```

Comment:
What you think about a base class for these collections? Look: https://github.com/pagseguro/ruby/pull/179/files#diff-5fc70af508af7ab6e557f26b8ab96a61R14